src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.cpp

```diff
@@ -96,8 +96,8 @@ namespace Util
     }
 
     void VisualizerMessenger::drawPoly(const std::string& layer,
-                                       std::vector<Point2D>& vertices,
-                                       DrawStyle draw_style, DrawTransform draw_transform)
+                                       std::vector<Point>& vertices, DrawStyle draw_style,
```

Comment:
This should be a `const std::vector<Point>&` otherwise we can accidentally modify the vertices.